tips/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from tips import views
+
+app_name = 'tips'
+urlpatterns = [
+    path('', views.board, name='board'),
+    path('tips/<int:tip_id>/', views.detail, name='detail'),
```

Comment:
tips/tips/id
that's a bit weird, I think you should change to a bit clear different URLs

Reply:
Will be fixed